main.go

```diff
@@ -0,0 +1,35 @@
+package main
+
+import (
+	"log"
+	"lolidelgado/github-users/controller"
+	"lolidelgado/github-users/delivery"
+	"lolidelgado/github-users/repository"
+	"lolidelgado/github-users/usecase"
+	"net/http"
+
+	"github.com/gorilla/mux"
+	"github.com/unrolled/render"
+)
+
+func main() {
+	const fileName = "github_users.csv"
+	//repository
+	githubUserRepo := repository.NewGithubUser(fileName)
+
+	//useCase
+	githubUserUseCase := usecase.NewGithubUser(githubUserRepo)
+
+	//controllers for Rest
+	httpRender := render.New()
+	usersController := controller.NewGithubUser(httpRender, githubUserUseCase)
+
+	httpRouter := mux.NewRouter()
+	delivery.Setup(
+		usersController,
+		httpRouter,
+	)
+
+	//start server
+	log.Fatal(http.ListenAndServe(":7000", httpRouter))
```

Comment:
I'd suggest to add a hint to the end user that the service is running.
```suggestion
	log.Info("Starting server at port 7000...")
	log.Fatal(http.ListenAndServe(":7000", httpRouter))
```

Reply:
Also, regarding the magic naming and magic numbers, in this case I'd also recommend to make the port a dynamic value; what if that port is already in use? you need to change your code base to amend that.

If you read that value from an external source you won't need that change in the code base.

Reply:
Updated, now the port is stored in an environment variable and can be updated if needed